### PR TITLE
Creating locale for pt-BR (Brazilian Portuguese)

### PR DESCRIPTION
**Pull request by ramirovjr**
With this commit the application will have a pt-BR locale
available for usage.

I've tested using the sandbox application pointing to my GH fork and it was ok.
Also, I've noticed that some interface items (like the 'Created' button) is not on the translation file.
Whenever you make changes or add new translation items, feel free to ping me for updating the translation.

### CODE REVIEW

**Comment by Linuus** [comment]
Awesome @ramirovjr Thank you very much.

Which "Created" button do you mean is not translatable?

**Comment by ramirovjr** [comment]
The one at the top of resource and I think it's 'Create' (sorry for posting the wrong label of the button)

**Comment by Linuus** [comment]
@ramirovjr I see. It is defined here:
https://github.com/varvet/godmin/blob/master/app/views/godmin/resource/_button_actions.html.erb

It uses the default translation from Rails, for creating models (`helpers.submit.create`), so it should pick it up from the regular locale file. We've never had the need to translate this separately so we just piggy backed on Rails. Would it be useful for you to be able to do that?

File: config/locales/pt-BR.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+pt-BR:
+  helpers:
+    submit:
+      create: Novo
```
Comment (Linuus) [inline config/locales/pt-BR.yml]: I don't think we should add any translations outside of the godmin scope. It will affect the entire site. If we want these to be translatable only in godmin we should add specific translations for them inside the godmin scope.

Can you revert these changes and squash it into one commit please?

**Comment by ramirovjr** [comment]
Yeah, no problem.
I'll do it tonight.

**Comment by Linuus** [comment]
@ramirovjr  I cherry-picked your commit directly instead :) So no need for squashing.

Thank you for the contribution!
